src/encoded/static/components/navigation.js

```diff
@@ -211,7 +211,7 @@ const Search = (props, context) => {
     const id = url.parse(context.location_href, true);
     const searchTerm = id.query.searchTerm || '';
     return (
-        <form className="navbar-form navbar-right" action="/search/">
+        <form className="navbar-form navbar-right navbar-search-form" action="/search/">
```

Comment:
I think instead of adding a class here, you could also have applied the flexbox styles to `navbar-form` because it is the only element on Encode with that class name.

Reply:
I can if you like. I did not originally because navbar-right is part of twitter bootstrap. I am normally hesitant to attach new style to 3rd party tools as it can lead to issues during version updates or in removal.

Reply:
Ah, yes, that is a good point! I agree that it is better with a new class name.